### PR TITLE
Add NoForwarding and EmptyForwardingPacketData helper function to use…

### PR DESCRIPTION
**Pull request by hakuno2000**
… in tests

## Description

closes: #6723

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against the correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to GitHub issue with discussion and accepted design, OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [conventional commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to follow the repository standards.
- [ ] Include a descriptive changelog entry when appropriate. This may be left to the discretion of the PR reviewers. (e.g. chores should be omitted from changelog)
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer.
- [ ] Review `SonarCloud Report` in the comment section below once CI passes.

### CODE REVIEW

**Review by crodriguezvega — APPROVED** [review]
Thank you for the PR, @hakuno2000! Changes look good to me; I pushed a commit using the new functions in a few more places.

**Comment by hakuno2000** [comment]
> Thank you for the PR, @hakuno2000! Changes look good to me; I pushed a commit using the new functions in a few more places.

Thank you so much sir.

File: modules/apps/transfer/types/forwarding.go [hunk 1]
```diff
@@ -18,6 +18,11 @@ func NewForwarding(unwind bool, hops ...Hop) *Forwarding {
 	}
 }
 
+// NoForwarding is a helper function which returns a Forwarding instance with all values empty.
+func NoForwarding() *Forwarding {
```
[the remark this answers is not in the document]
Comment (crodriguezvega) [reply modules/apps/transfer/types/forwarding.go:22]: Makes sense. I was updating it now...

File: modules/apps/transfer/internal/packet_test.go [hunk 2]
```diff
@@ -43,7 +41,7 @@ func TestUnmarshalPacketData(t *testing.T) {
 							Denom:  types.NewDenom("atom", types.NewHop("transfer", "channel-0")),
 							Amount: "1000",
 						},
-					}, sender, receiver, "", emptyForwardingPacketData)
+					}, sender, receiver, "", types.ForwardingPacketData{})
```
Comment (crodriguezvega) [inline modules/apps/transfer/internal/packet_test.go:44]: I couldn't use the testing value because it was complaining about some import cycle...

Comment (crodriguezvega) [reply modules/apps/transfer/internal/packet_test.go:44]: Yeah, I also was surprised, but decided to skip this rabbit hole...

**Pull request merged**